apps/dapp-console-api/src/models/contracts.ts

```diff
@@ -70,3 +79,13 @@ export const getContractsForApp = async (input: {
     .where(and(eq(contracts.appId, appId), eq(contracts.entityId, entityId)))
     .orderBy(asc(contracts.createdAt))
```

Comment:
nit: is the entityId filter necessary? I thought an app always has a single entityId associated with it

Reply:
I added this as a security measure. This makes it so that a client fetching a contract needs to provide the entity id associated with that contract. without this a client associated with a different entity could fetch a contract by just knowing the contract id of another entity's contract